CONTRIBUTING.md

```diff
@@ -22,12 +22,23 @@ And run the test suite (should be all green with pending tests):
 
 If you have problems installing nmatrix, please consult the [nmatrix installation wiki](https://github.com/SciRuby/nmatrix/wiki/Installation) or the [mailing list](https://groups.google.com/forum/#!forum/sciruby-dev).
 
+**NOTE**: `Daru` is compatible with Ruby versions < 2.5; for later Ruby versions it breaks, returning the following error when `bundle exec rspec` is run for Ruby 
```

Comment:
In Issue https://github.com/SciRuby/daru/issues/500, it was breaking while running 

```
df = Daru::DataFrame.new(a: [1,2,3], b:[4,5,6])
df.each(:row) { |i| puts i }
```

Reply:
I am guessing this error appears whenever Daru is built using Ruby>2.5? I wanted to include the part here because it's part of the process for setting up Daru. But I guess there are a lot of other instances/commands where the build will eventually fail and return the same error message and we can't document them all here. Will remove the part for now.

Reply:
> I am guessing this error appears whenever Daru is built using Ruby>2.5?

Yes!



> But I guess there are a lot of other instances/commands where the build will eventually fail and return the same error message and we can't document them all here. Will remove the part for now.

I am not aware of those cases. Please let us know if you have in mind. It will help to fix and track it further.